src/server/code-parse/language/default/codecrumbs.js

```diff
@@ -57,22 +53,21 @@ const buildCrumb = (params, crumbNodeLines, path) => ({
 });
 
 const setupGetCommentsFromCode = regex => fileCode => {
-  if (!fileCode) return [];
-
-  return fileCode.split('\n').reduce((comments, item, i) => {
-    const codeLine = item.trim();
-    if (!codeLine) return comments;
-
-    const matches = regex.exec(codeLine);
-    if (matches) {
-      const lineNumber = i + 1;
-      return [
-        ...comments,
-        { value: matches[matches.length - 1], nodeLines: [lineNumber, lineNumber] }
-      ];
-    }
+  if (!fileCode) {
+    return [];
+  }
+
+  const result = fileCode.match(regex) || [];
```

Comment:
@idoo match returns more than you need :)
if you run ```yarn server:two``` it will look like this
![image](https://user-images.githubusercontent.com/2098309/62415463-4f7d3c80-b62a-11e9-9f4c-4c47649f088c.png)
while in master it's like this
![image](https://user-images.githubusercontent.com/2098309/62415468-5a37d180-b62a-11e9-9bb7-4763acdf030c.png)
you can see there is an issue with example-project/src-server/app/service/user/session.py 

Match returns 
```
[ '#cc:signin#3;updare seesion',
  '',
  'cc:signin#3;updare seesion',
  index: 0,
  input: '#cc:signin#3;updare seesion' ]
```
there, so after filer by //cc you gonna have double '#cc:signin#3;updare seesion'